frontend/src/api.js

```diff
@@ -0,0 +1,82 @@
+function checkStatus(resp) {
```

Comment:
why not moving this service into its own place?
In `cat` we used `/api` package, for example.

Reply:
What would be the benefit of moving `api.js` to `/api/index.js`? I don't have a strong opinion on this, let me know why and we can move it.

Reply:
I see two reasons;

first one is kindof opinionated: imo the root dir should be populated with as fewer things as possible because it helps to understand better the structure of the code.

second one is generic: now we have in this file the logic (copy-pasted from `cat`) to connect with the API (business agnostic), and also the "repository-like" code (business related). Splitting both parts properly, and moving business agnostic to a service, and the business logic to a repository would help us to reuse and test better.